src/index.html

```diff
@@ -7,8 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <body>
+      <header class="header">
+        <a href="#home">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#Apple">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#Samsung">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#Smartphones">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                data-qa="hover"
+                href="#Laptops & Computers">Laptops & Computers
```

Comment:
https://mateacademy.slack.com/archives/C050SHYFFR9/p1681912970291919

Reply:
is that ccorrect now? https://prnt.sc/qxL615uHgH3e